src/test/resources/pages/studentFeedbackSubmitPageOpen.html

```diff
@@ -3905,26 +3905,26 @@ <h4 class="modal-title">
             <div class="col-sm-12">
               <table>
                 <tbody>
-                <tr>
-                  <td>
-                    <div class="radio">
-                      <label class="bold-label">
-                        <input name="responsetext-26-0" type="radio" value="Team 2">
-                        Team 2
-                      </label>
-                    </div>
-                  </td>
-                </tr>
-                <tr>
-                  <td>
-                    <div class="radio">
-                      <label class="bold-label">
-                        <input name="responsetext-26-0" type="radio" value="Team 3">
-                        Team 3
-                      </label>
-                    </div>
-                  </td>
-                </tr>
+                  <tr>
```

Comment:
Are these indentation changes needed for this PR?
Reverting such changes should not break the PR if I am not wrong

Reply:
It's not needed for this PR! The change was caught by GodMode. 

Reverted the change for all similar instances :smile:

Reply:
Not your fault. It was an existing problem which was found now #8744

Reply:
@tshradheya I just realized reverting the changes made to this 2 files do break the PR. 

I tried to manually add the line `<input checked="" name="sendsubmissionemail" type="checkbox">` in the 2 files (the necessary changes minus the indentation) but it showed this error instead:

```
#### Incorrect page type: going to try reloading the page.
######### Not in the correct page! ##########
```

Reply:
This was the commit in which you added the `<input checked="" name="sendsubmissionemail" type="checkbox">` 
https://github.com/TEAMMATES/teammates/pull/8742/commits/a5ac5db24a507fa8b128c62f1ec045d037b82e19

And this was the commit in which you reverted all the godmode changes https://github.com/TEAMMATES/teammates/pull/8742/commits/546774f14062342345d528880d99b0d4f3c49f27

There are some places where you forgot to update `<input name="sendsubmissionemail" type="checkbox">` to `<input checked="" name="sendsubmissionemail" type="checkbox">` E.g. 
- unregisteredStudentFeedbackSubmitPageOpen.html

On updating those the problem should be solved